test/api.ts

```diff
@@ -228,10 +230,10 @@ tape('[Transaction]: Basic functions', function(t) {
       '4646464646464646464646464646464646464646464646464646464646464646',
       'hex',
     )
-    const pt = new Transaction(txRaw, { chain: 1 })
+    const pt = new Transaction(txRaw)
     st.equal(
       pt.serialize().toString('hex'),
-      'ec098504a817c800825208943535353535353535353535353535353535353535880de0b6b3a764000080018080',
+      'ec098504a817c800825208943535353535353535353535353535353535353535880de0b6b3a764000080808080',
```

Comment:
Yes, it's because of that. 

Note that what the EIP calls "signing data" isn't the output of `serialize()`, it's the preimage of the hash returned by `hash(false)`. We don't have a getter for such value in this class.

Reply:
I added an explanation about this in the test